.github/workflows/rust.yml

```diff
@@ -288,17 +318,20 @@ jobs:
           mv *.tbl ../datafusion/sqllogictest/test_files/tpch/data
       - name: Verify that benchmark queries return expected results
         run: |
+          # increase stack size to fix stack overflow
+          export RUST_MIN_STACK=20971520
```

Comment:
I think we should have fixed this now with the recursive protection feature -- do we still need to set the min stack size?

Reply:
I know I needed it when I tried running this the other day. When was that fixed?

Reply:
It was in https://github.com/apache/datafusion/pull/13310